domains/zplusfour.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "zplusfour's personal developer website",
+  "repo": "https://github.com/zplusfour",
+  "owner": {
+    "username": "zplusfour",
+    "email": "zgjoniur@gmail.com"
+  },
+  "record": {
+    "CNAME": "zplusfour.repl.co"
```

Comment:
Refer to the custom domains section of https://docs.replit.com/repls/web-hosting for information regarding custom domains with replit

Reply:
;-;